test_crates/item_missing/new/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "item_missing"
```

Comment:
`item_missing` tests for multiple different kinds of items going missing. Would it be better to have separate crates for different kinds of items?